src/pybind/mgr/orchestrator/tests/test_orchestrator.py

```diff
@@ -191,11 +191,15 @@ def test_orch_ps(_describe_service):
     }
     m = OrchestratorCli('orchestrator', 0, 0)
     r = m._handle_command(None, cmd)
-    out = 'NAME    HOST       PORTS  STATUS   REFRESHED  AGE  MEM USE  MEM LIM  VERSION    IMAGE ID   \n'\
-          'osd.1   <unknown>         unknown          -    -        -        -  <unknown>  <unknown>  \n'\
-          'osd.2   <unknown>         unknown          -    -        -        -  <unknown>  <unknown>  \n'\
-          'osd.10  <unknown>         unknown          -    -        -        -  <unknown>  <unknown>  '
-    assert r == HandleCommandResult(retval=0, stdout=out, stderr='')
+    expected_out = 'NAME    HOST       PORTS  STATUS   REFRESHED  AGE  MEM USE  MEM LIM  VERSION    IMAGE ID   \n'\
+                   'osd.1   <unknown>         unknown          -    -        -        -  <unknown>  <unknown>  \n'\
+                   'osd.2   <unknown>         unknown          -    -        -        -  <unknown>  <unknown>  \n'\
+                   'osd.10  <unknown>         unknown          -    -        -        -  <unknown>  <unknown>  '
+    expected_out = [c for c in expected_out if c.isalpha()]
+    actual_out = [c for c in r.stdout if c.isalpha()]
```

Comment:
Seems OK, but maybe next time try a lib like https://docs.python.org/3/library/difflib.html  which has functions to help you ignore whitespace?  I think the output of difflib (or something simlar) may be more readable too. I think difflib is a depenedency of pytest so it may already be available in all places we need it...

Reply:
Interesting. I wasn't aware of this library. I'll have to play around with it a bit and see if I can get something that works for this case.

Reply:
Meh, I thought it would be a simple one liner but after trying a few things I didn't find anything worth replacing your simple version with. Ignore this suggestion for now.